content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,79 +1,121 @@
-===================================================
-Use product variants on quotations and sales orders
-===================================================
+===============================================
+Product variants on quotations and sales orders
+===============================================
 
-Product variants can be added to quotations and sales orders using the Product Configurator,
-Variant Grid Entry, or both.
+Before getting into detail about how to use product variants on quotations and sales orders, it's
+recommended to learn about :doc:`/applications/sales/sales/products_prices/products/variants` in
+Odoo.
 
-Configuration
-=============
+Once familiarized with the basics surrounding product variants, the following covers how product
+variants can be added to quotations and sales orders using the *Product Configurator* or *Order Grid
+Entry*.
 
-To enable product variants on quotations and sales orders, go to :menuselection:`Sales -->
-Configuration --> Settings --> Product Catalog` and activate one or both of the
-:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally, click
-:guilabel:`Save` to apply the settings.
+.. note::
+   It should be noted that the setting is titled, `Variant Grid Entry` on the *Sales* app settings
+   page, and titled, `Order Grid Entry` on product forms. So, be sure to keep that in mind.
+
+Settings
+========
+
+When working with product variants, Odoo uses the :guilabel:`Product Configurator`, by default. To
+add the :guilabel:`Variant Grid Entry` option, that feature **must** be enabled in the Odoo *Sales*
+application.
```

Comment:
A brief explanation about why we'd want to add the Variant Grid Entry option here would help readers decide whether they want to add this feature :open_mouth: